pkg/provider/provider.go

```diff
@@ -44,12 +49,8 @@ var (
 	loaded = false
 )
 
-func GetContainer(namespace, podName, containerName string) (v1.Container, error) {
-	return v1.Container{}, nil
-}
-
-func GetPod(namespace, podName string) (v1.Pod, error) {
-	return v1.Pod{}, nil
+func GetContainer() *Container {
```

Comment:
the intent of this function was to have a quick access to a specific container without looping through them. The code was just a function declaration without any implementation